### PR TITLE
Marketplace ContentAPI changes

### PR DESCRIPTION
**Pull request by serdarceranmendix**
Frst get approval from internal stakeholders (@KseniiaIankina)
DONE

### CODE REVIEW

**Comment by katarzyna-koltun-mx** [comment]
@Luyao-Zhang-1 let me know if the PR should be reassigned.

File: content/en/docs/apidocs-mxsdk/apidocs/content-api.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+url: /apidocs-mxsdk/apidocs/content-api/
+type: swagger
+category: "API Documentation"
+description: "This API allows you to access both public and company-specific Marketplace content information."
+weight: 47
+tags: ["API", "content"]
+---
+
+{{% alert color="info" %}}
+This API is only available via a private beta program. If you want to be part of this program, kindly reach out to [<u>Marketplace team</u>](mailto:Marketplace_Team@Mendix.com) to get access.
```
Comment (ConnorLand) [inline content/en/docs/apidocs-mxsdk/apidocs/content-api.md]: @Luyao-Zhang-1 I have tested in a local build, and these underline divs do work successfully. However, I think underline styling is not needed. The regular way our CSS makes hyperlink text pop will draw attention to the link quite well.

Comment (serdarceranmendix) [reply content/en/docs/apidocs-mxsdk/apidocs/content-api.md]: Hi @ConnorLand ,
No, underline styling on "Marketplace team" email link is needed indeed.

> underline styling is not needed

Comment (KseniiaIankina) [reply content/en/docs/apidocs-mxsdk/apidocs/content-api.md]: Hi all, @ConnorLand if it puts attendtion that it is a link, then great, we do not need it underlined. Thank you

**Comment by serdarceranmendix** [comment]
Hi @katarzyna-koltun-mx and @Luyao-Zhang-1,
As we are still in the beta program phase, we have decided to postpone the release of the documentation by one month. We believe it's essential to refine our API based on the feedback that we will receive during the beta program before we finalize and publish the documentation.

Is it also fine for you?

**Comment by Adam-Dupaski** [comment]
Hi @serdarceranmendix yes, if the API is currently only in Private Beta, we can wait to publish until it is Public Beta or GA, that is fine.

File: static/openapi-spec/marketplace-content.yaml [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+          type: string
+          format: url
+          example: "http://www.apache.org/licenses/LICENSE-2.0.html"
+          description: "The license selected during onboarding."
+        isPrivate:
+          type: boolean
+          description: "This indicates whether the content is public or private."
+        latestVersion:
+          $ref: '#/components/schemas/version'
+    content:
```
Comment (Luyao-Zhang-1) [inline static/openapi-spec/marketplace-content.yaml:322]: @serdarceranmendix  Do you think we should add "description" to "content"?

File: static/openapi-spec/marketplace-content.yaml [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+      type: object
+      properties:
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/specificContent'
+    contentVersions:
```
Comment (Luyao-Zhang-1) [inline static/openapi-spec/marketplace-content.yaml:330]: @serdarceranmendix Do you think we should add "description" to "ContentVersions"?

File: static/openapi-spec/marketplace-content.yaml (hunk 2 above)
Comment (serdarceranmendix) [reply static/openapi-spec/marketplace-content.yaml:322]: I just added for 'content'. can you please check @Luyao-Zhang-1 ?

File: static/openapi-spec/marketplace-content.yaml (hunk 3 above)
Comment (serdarceranmendix) [reply static/openapi-spec/marketplace-content.yaml:330]: The description is not needed for this one. @Luyao-Zhang-1 .

**Pull request merged**